recipes/wot/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "wot" %}
+{% set version = "1.0.8.post2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wot-{{ version }}.tar.gz
+  sha256: 89f2a4440c9516d66bc453f4f50ed81d5c33f1223f0f927f9e35a2c9e875bfb9
+
+build:
+  entry_points:
+    - wot=wot.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.0
+    - setuptools-scm
+    - cython
```

Comment:
My guess is that cython is for optional speedups or a left over from a re-factor, right?